roles/key_inject/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+subkey_path: 'https://s3.eu-central-1.amazonaws.com/releases.parity.io/substrate/x86_64-debian%3Astretch/v3.0.0/subkey/subkey'
```

Comment:
```suggestion
subkey_path: 'https://releases.parity.io/substrate/x86_64-debian%3Astretch/v3.0.0/subkey/subkey'
```

Reply:
We can just delete it, it is not used (it was used in old version)